Bottom: 69f2ed81724eb57357d3d1d52d18d9995504f53e
Top:    f03442193562ba2aa2dcff92e118eecb964be233
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 11:32:51 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 92c8b35..765aeec 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2747,7 +2747,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
